servers/physics_2d/body_pair_2d_sw.cpp

```diff
@@ -257,7 +257,7 @@ bool BodyPair2DSW::setup(real_t p_step) {
 	}
 	//faster to set than to check..
 
```

Comment:
While we're about it, I would delete this comment that is not relevant.

Reply:
Pushed an amend to do so.